Corrected scope fitting to resolve issue #94.

ksmith0 opened the pull request:
The fitting function was mistreating the portion when diff was less than
zero. This should have returned the baseline.

In addition, removed any parameter limits and added documentation for
the function.

Commit:
Corrected scope fitting to resolve issue #94.

The fitting function was mistreating the portion when diff was less than
zero. This should have returned the baseline.

In addition, removed any parameter limits and added documentation for
the function.

wapmanzoo reviewed (commented):
Making a comment to start the review. Now trying to find how to pull the code to test it.

spaulaus committed:
Fixing the value of the baseline for the fitting

The baseline value should be fixed since the fit in the scan works on baseline subtracted traces. This change is confirmed to give similar results to gnuplot fits of the same trace.

spaulaus reviewed (approved):
Now that the baseline is fixed, fit results are identical to other software.
